src/main.ts

```diff
@@ -4,15 +4,26 @@ console.log('Hello from typescript')
 const input = document.querySelector<HTMLInputElement>('#todo-input')
 const output = document.querySelector('.todo-element')
 const button = document.querySelector('#add-todo-button')
+const todos: string[] = []
 
 function addtolist(): void {
```

Comment:
On the first load of the page, you need to show the todos already present in local storage.